extensions/binding/org.eclipse.smarthome.binding.iota/README.md

```diff
@@ -0,0 +1,61 @@
+# IOTA Binding
+
+This binding allows to bind a IOTA topic to a Thing. A bridge connection needs to be already defined, i.e you first need to create an IOTA bridge thing.
+The way this binding works is heavily inspired by the MQTT generic binding that maps JSON data to channels.
+
+## Supported Bridge
+
+* **IOTA BRIDGE**: This bridge represents an instance that connects to an IOTA node. 
+
+## Supported Thing
+
+There is one thing available ("topic"), where you can add the following channels to:
+
+## Supported Channels
+
+* **text**: This channel can show the received text on the given topic.
+* **number**: This channel can show the received number on the given topic. It can have a min, max and step values.
+* **percentage**: This channel handles numeric values as percentages. It can have a min, max and step values.
+* **onoff**: This channel represents a on/off state of a given topic.
+
+## Thing and Channel configuration
+
+The bridge only needs a **protocol**, **host** and **port**. A thing offers a few parameters for the configuration: the frequence at which it refreshes the data, the mode it uses for MAM, and the root address from which to start fetching the data. 
+
+All thing channels support JSON/XML unpacking: Usually a IOTA topic state represents a plain value like a text or a number.
+
+### Common channel configuration parameters
+
+* __stateTopic__: The IOTA topic that represents the state of the thing. For instance, if you're tracking a sensor that sends humidity related data on IOTA's distributed ledger through the io.iota bundle, the stateTopic will be **humidity**. 
+
+### Work In Progress:
+
+* __transformationPattern__: An optional transformation pattern like [JSONPath](http://goessner.net/articles/JsonPath/index.html#e2). Use http://jsonpath.com/ to verify your pattern for the latter case. An example for a received JSON from a IOTA state topic would be a pattern of JSONPATH:$.thingNumber.status.state for a json `[{"NAME": name, "STATUS": { "TOPIC": topic, "STATE": state, "TIME": time }}]` to extract the temperature value.
```

Comment:
in the end this should be "done", not WIP :-)

Reply:
Will change that when we will all have agreed on the modeling 👍